WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/UserNoteBlock.java

```diff
@@ -87,16 +89,25 @@ public View configureView(View view) {
             String imageUrl = GravatarUtils.fixGravatarUrl(getNoteMediaItem().optString("url", ""), getAvatarSize());
             noteBlockHolder.avatarImageView.setImageUrl(imageUrl, WPNetworkImageView.ImageType.AVATAR);
             if (!TextUtils.isEmpty(getUserUrl())) {
+                noteBlockHolder.avatarImageView.setOnClickListener(new View.OnClickListener() {
```

Comment:
I just want to confirm, that we really need to set the onClickListener to the avatarImageView as we set an onClickListener to the root view few lines below. If we need it, we could probably reuse the `mOnClickListener` field.

Reply:
Yes, you are totally right. I just focused on gravatar, without realizing that tap on root view leads to the same place.